src/components/screens/monitor/transactions/transactions.js

```diff
@@ -1,26 +1,61 @@
 import React from 'react';
-import grid from 'flexboxgrid/dist/flexboxgrid.css';
 import MonitorHeader from '../header';
 import TransactionsTable from '../../../shared/transactionsTable';
 
 const Transactions = ({ t, transactions }) => {
-  const columns = [
-    { header: t('Sender'), className: `${grid['col-xs-3']}`, key: 'senderId' },
-    { header: t('Recipient'), className: `${grid['col-xs-3']}`, key: 'recipientId' },
-    {
-      header: t('Date'), className: `${grid['col-xs-2']}`, key: 'timestamp', isSortingColumn: true, defaultSort: true,
-    },
-    {
-      header: t('Amount'), className: `${grid['col-xs-2']}`, key: 'amount', isSortingColumn: true,
-    },
-    { header: t('Fee'), className: `${grid['col-xs-1']}  hidden-m`, key: 'fee' },
-    { header: t('Status'), className: `${grid['col-xs-1']}`, key: 'confirmations' },
-  ];
+  const fields = [{
+    label: t('Date'),
+    name: 'date',
+    type: 'date-range',
+  }, {
+    label: t('Amount Range'),
+    name: 'amount',
+    type: 'number-range',
+  }, {
+    label: t('Sender'),
+    placeholder: 'Address or Public key',
+    name: 'sender',
+    type: 'address',
+  }, {
+    label: t('Message'),
+    placeholder: t('Your message'),
+    name: 'message',
+    type: 'text',
+    // TODO: Enable field when API allows message param
+    disabled: true,
```

Comment:
The disabled field without any explanation feels broken. I would comment out the whole object here.